_posts/en/newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,205 @@
+---
+title: 'Bitcoin Optech Newsletter #241'
+permalink: /en/newsletters/2023/03/08/
+name: 2023-03-08-newsletter
+slug: 2023-03-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Alternative design for OP_VAULT:** Greg Sanders [posted][sanders
+  vault] to the Bitcoin-Dev mailing list an alternative design for
+  providing the features of the `OP_VAULT`/`OP_UNVAULT` proposal (see
+  [Newsletter #234][news234 vault]).  His alternative would add three
+  opcodes instead of two.  To provide an example:
+
+    - *Alice deposits funds in a vault* by paying a [taproot][topic
+      taproot] tree which contains at least two [tapscripts][topic
+      tapscript], one with her unvault tapscript and one with her freeze
+      tapscript, e.g. `tr(key,{trigger,freeze})`.
+
+      - The *trigger tapscript* contains her less-trusted authorization
+        conditions (such as requiring a signature from her hot wallet)
+        and an `OP_TRIGGER_FORWARD` opcode.  At the time she creates
+        this tapscript, she provides the opcode a *spend delay*
+        parameter, e.g. a relative timelock of 1,000 blocks (about 1
+        week).
+
+      - The *freeze tapscript* contains any authorization conditions
+        Alice wants to specify (including none at all) and
+        an `OP_FORWARD_DESTINATION` opcode.  At the time she creates
+        this tapscript, she also chooses her more-trusted authorization
+        conditions (such as requiring multiple signatures from multiple
+        cold wallets and hardware signing devices).  She provides the
+        opcode a commitment to those conditions in the form of a hash
+        digest.
+
+    - *Alice initiates an unvaulting* by spending the output received to
+      the above taproot tree (using it as an input) and choosing the
```

Comment:
Since the script tree is only a subset of the spending conditions in a P2TR output, the “received to the taproot tree” feels a bit odd to me. 

```suggestion
      the output with the above script tree (using it as an input) and choosing the
```